.golangci.yml

```diff
@@ -62,6 +64,18 @@ linters-settings:
     forbid-focus-container: true
   gofumpt:
     extra-rules: true
+  gci:
+    custom-order: true
+    sections:
+    - standard
+    - default
+    - prefix(github.com/projectcontour/contour)
+  importas:
+    alias:
```

Comment:
Yeah, we should try to get the core k8s APIs added here as well for consistency, but fine to do that as a follow-up.